src/main/java/com/collector/config/ThymeleafConfig.java

```diff
@@ -0,0 +1,51 @@
+package com.collector.config;
+
+import javax.servlet.ServletContext;
+import javax.servlet.ServletContextEvent;
+import javax.servlet.ServletContextListener;
+import javax.servlet.annotation.WebListener;
+import org.thymeleaf.TemplateEngine;
+import org.thymeleaf.templatemode.TemplateMode;
+import org.thymeleaf.templateresolver.ITemplateResolver;
+import org.thymeleaf.templateresolver.ServletContextTemplateResolver;
+
+/**
+ * Thymeleaf configuration.
+ */
+@WebListener
+public class ThymeleafConfig implements ServletContextListener {
+
+    public void contextInitialized(ServletContextEvent sce) {
+        TemplateEngine engine = templateEngine(sce.getServletContext());
+        TemplateEngineUtil.storeTemplateEngine(sce.getServletContext(), engine);
+    }
+
+    public void contextDestroyed(ServletContextEvent sce) {
+    }
+
+    private TemplateEngine templateEngine(ServletContext servletContext) {
+        TemplateEngine engine = new TemplateEngine();
+        engine.setTemplateResolver(templateResolver(servletContext));
+        return engine;
+    }
+
+//    private TemplateEngine javascriptTemplateEngine(ServletContext servletContext) {
+//        TemplateEngine engine = new TemplateEngine();
+//        engine.setTemplateResolver(javascriptTemplateResolver(servletContext));
+//        return engine;
+//    }
+
+    private ITemplateResolver templateResolver(ServletContext servletContext) {
```

Comment:
toto ma byt len TemplateResolver alebo co tam znamena to I?

Reply:
Nazov triedy timeleaf. Zrejme v minulosti pouzivali TemplateResolver ktory je teraz deprecated tak potrebovali novy nazov.